chat-plugins/health-fitness.js

```diff
@@ -0,0 +1,147 @@
+			);
+		},
+		cardio: function (target, room, user) {
+			if (room.id !== 'healthfitness') return this.errorReply("This command can only be used in Health & Fitness.");
+			if (!target) {
+				if (!this.canBroadcast('!healthfitness cardio')) return;
+				if (this.broadcastMessage && !this.can('broadcast', null, room)) return false;
+
+				if (!this.runBroadcast('!healthfitness cardio')) return;
+
+				const cardio = HF.hfData.cardio || '(not set yet)';
+				return this.sendReplyBox(Chat.html`<strong>Cardio Challenge:</strong> ${cardio}`);
+			} else {
+				if (!this.can('broadcast', null, room)) return;
+				if (target.length > MAX_DATA_LENGTH) return this.errorReply(`This challenge is too long; it cannot exceed ${MAX_DATA_LENGTH} characters.`);
+				HF.setCardio(target.trim());
+				this.privateModCommand(`(${user.name} has set the cardio challenge to: ${target})`);
+				this.sendReply(`The daily cardio challenge has been updated to: ${target}`);
+			}
+		},
+		gym: function (target, room, user) {
+			if (room.id !== 'healthfitness') return this.errorReply("This command can only be used in Health & Fitness.");
+			if (!target) {
+				if (!this.canBroadcast('!healthfitness gym')) return;
+				if (this.broadcastMessage && !this.can('broadcast', null, room)) return false;
+
+				if (!this.runBroadcast('!healthfitness gym')) return;
+
+				const gym = HF.hfData.gym || '(not set yet)';
+				return this.sendReplyBox(Chat.html`<strong>Gym Challenge:</strong> ${gym}`);
+			} else {
+				if (!this.can('broadcast', null, room)) return;
+				if (target.length > MAX_DATA_LENGTH) return this.errorReply(`This challenge is too long; it cannot exceed ${MAX_DATA_LENGTH} characters.`);
+				HF.setGym(target.trim());
+				this.privateModCommand(`(${user.name} has set the gym challenge to: ${target})`);
+				this.sendReply(`The daily gym challenge has been updated to: ${target}`);
+			}
+		},
+		quote: function (target, room, user) {
+			if (room.id !== 'healthfitness') return this.errorReply("This command can only be used in Health & Fitness.");
+			if (!target) {
+				if (!this.canBroadcast('!healthfitness quote')) return;
+				if (this.broadcastMessage && !this.can('broadcast', null, room)) return false;
+
+				if (!this.runBroadcast('!healthfitness quote')) return;
+
+				const quote = HF.hfData.quote || '(not set yet)';
+				return this.sendReplyBox(Chat.html`<strong>Quote of the Day:</strong> ${quote}`);
+			} else {
+				if (!this.can('broadcast', null, room)) return;
+				if (target.length > MAX_DATA_LENGTH) return this.errorReply(`This quote is too long; it cannot exceed ${MAX_DATA_LENGTH} characters.`);
+				HF.setQuote(target.trim());
+				this.privateModCommand(`(${user.name} has set the daily quote to: ${target})`);
+				this.sendReply(`The daily quote has been updated to: ${target}`);
+			}
+		},
+	},
+	healthfitnesshelp: [
+		"/healthfitness - Shows the daily cardio challenge, gym challenge, and quote of the day",
```

Comment:
this day is missing a period after it. the lines that include "or above" also appear to be missing a period unless that's intentional.